src/engine/QueryExecutionTree.h

```diff
@@ -180,6 +194,19 @@ class QueryExecutionTree {
                     row[validIndices[validIndices.size() - 1].first]))
              << "\"]";
           break;
+        case ResultTable::ResultType::FLOAT: {
+          float f;
+          std::memcpy(&f, &row[validIndices[validIndices.size() - 1].first],
```

Comment:
I think at another point something similar used a `reinterpret_cast` I think this variant here is cleaner and safer

Reply:
I replaced the reinterpret casts with memcpys. The standard seems to actually not define any behaviour for the reinterpret casts when used for non similar types, so this should be more standard conform.

Reply:
Yes I think that's why I had it on my mind that it's possibly safer